ota-plus-web/app/reactapp/package.json

```diff
@@ -65,6 +65,7 @@
     "extract-loader": "3.1.0",
     "file-loader": "2.0.0",
     "html-loader": "0.5.5",
+    "html-webpack-plugin": "^3.2.0",
```

Comment:
Remove dash

Reply:
Actually, we don't need this since the favicon is injected in scala html template, removed.